### DISCUSSION TITLE
GO ora CLOB error y compatibilidad con sqlx

### DISCUSSION THREAD

**Discussion by rafael180496** [sijms/go-ora#76] [Q&A]
Esta dando un problema la version 2 con un select 

 SELECT NOM_USR,CLAVE
        FROM ELSV_USUARIO_FEL
        WHERE ROWNUM <2
        AND COMPANY=:COMPANY

donde :company va con la sub lirberia sqlx 

https://github.com/jmoiron/sqlx

el error es caracter invalid

**Comment by sijms** [comment]
can you explain the problem more clearly
what is the data type of the Columns?
What is your database charset?
does this error occur with and without sqlx or only with sqlx?
